en/contributing/build.md

```diff
@@ -124,16 +152,6 @@ build/default/integration_tests_runner --gtest_filter="ExternalExampleHello"
 ```
 
 
-
-### Code Style
-
-All cpp and h files should be formatted according to the astyle settings defined by astylerc.
-To automatically fix the formatting, run this command:
-
-```
-make fix_style
-```
-
 ## Building in Docker
```

Comment:
We could also use the docker build to generate wrappers, etc. It's just a build environment if you don't want to mess with your own.

If you want to connect through docker, you would have to forward the UDP port, e.g. for SITL:

```
-p 14540:14540/udp -p 
```

So the full docker command would be:
```
docker run -it -p 14540:14540 -v $HOME/src/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
```

And in there you can run:
```
make && build/default/integration_tests_runner --gtest_filter="SitlTest.ActionAsyncHover"
```